Regression_test/CMakeLists.txt

```diff
@@ -3,26 +3,35 @@ project(Regression_test LANGUAGES CXX)
 
 include(cmake/environment.cmake)
 
+find_package(Sofa.Simulation.Graph REQUIRED)
+sofa_find_package(Sofa.Testing REQUIRED)
```

Comment:
In the PR description, you said you don't need it anymore

Reply:
I use Sofa.Testing to get gtest 🥸